Tests/Marketplace/core_packs_list.json

```
@@ -1,34 +1,35 @@
 [
-  "Base",
-  "rasterize",
-  "DemistoRESTAPI",
-  "DemistoLocking",
-  "Malware",
-  "ImageOCR",
-  "AutoFocus",
-  "UrlScan",
   "Active_Directory_Query",
-  "FeedTAXII",
-  "VirusTotal",
-  "Whois",
-  "CommonScripts",
-  "CommonPlaybooks",
-  "CommonTypes",
+  "AutoFocus",
+  "Base",
   "CommonDashboards",
+  "CommonPlaybooks",
   "CommonReports",
+  "CommonScripts",
+  "CommonTypes",
   "CommonWidgets",
+  "CortexXDRIR",
+  "DefaultPlaybook",
+  "DemistoLocking",
+  "DemistoRESTAPI",
+  "EDL",
+  "FeedMitreAttackv2",
+  "FeedTAXII",
+  "FeedUnit42v2",
+  "FiltersAndTransformers",
+  "HelloWorld",
+  "ImageOCR",
+  "Palo_Alto_Networks_WildFire",
+  "Panorama",
+  "Redlock",
```

Didn't you mean this?
```suggestion
  "PAN-OS",
  "PrismaCloud",
```

---

Should it be the directory name? if so you are correct.